src/atlclients/clientManager.ts

```diff
@@ -216,10 +216,17 @@ export class ClientManager implements Disposable {
         );
     }
 
-    private createBasicHTTPClient(site: DetailedSiteInfo, username: string, password: string): HTTPClient {
+    private createHTTPClient(site: DetailedSiteInfo, info: AuthInfo): HTTPClient {
+        let auth = '';
+        if (isBasicAuthInfo(info)) {
+            auth = `Basic ${Buffer.from(info.username + ':' + info.password).toString('base64')}`;
+        } else if (isPATAuthInfo(info)) {
+            auth = `Bearer ${info.token}`;
+        }
+        Logger.info(auth);
```

Comment:
Logging Auth is probably not the right thing to do. 

Instead, we could log what type of auth we are seeing.